### DISCUSSION TITLE
the data annotation file ):

### DISCUSSION THREAD

**Discussion by haifa33** [cabani/MaskedFace-Net#11] [General]
please, is there a data annotation file for the proposed dataset? 
and if it is an xml file, it will be a great help

thank you

**Comment by cabani** [comment]
Dear @haifa33,

Each image of the MaskedFace-Net dataset has its corresponding file naming in the FFHQ dataset. Hence, the metadata file “ffhq-dataset-v2.json” (see the FFHQ webpage) can be exploited for your processing of MaskedFace-Net.

Regards,
Maskedface-Net Team
